configure.ac

```diff
@@ -192,17 +192,7 @@ AC_ARG_WITH(bubblewrap,
               [with_bubblewrap=/usr/bin/bwrap])
 AC_DEFINE_UNQUOTED(WITH_BUBBLEWRAP_PATH, ["$with_bubblewrap"], [Define to bubblewrap path])
 
-dnl Some distributions may want to support the client tooling, but not
-dnl the server side.
-AC_ARG_ENABLE(compose-tooling,
-              AS_HELP_STRING([--enable-compose-tooling],
-                             [Build compose tooling]),,
-              [enable_compose_tooling=yes])
-AM_CONDITIONAL(BUILDOPT_COMPOSE_TOOLING, [test x$enable_compose_tooling = xyes])
-AS_IF([test x$enable_compose_tooling = xyes], [
-    AC_DEFINE(HAVE_COMPOSE_TOOLING, 1, [Define if we are building compose tooling])
-])
-if test x$enable_compose_tooling != xno; then RPM_OSTREE_FEATURES="$RPM_OSTREE_FEATURES compose"; fi
+RPM_OSTREE_FEATURES="$RPM_OSTREE_FEATURES compose"
```

Comment:
Minor: might as well just not include it in the list of features if it's always baked.
Actually, rust is also a hard req now, so we could just empty out features altogether. And maybe also

```diff
diff --git a/src/app/main.c b/src/app/main.c
index 0a222619..6c6ad5d7 100644
--- a/src/app/main.c
+++ b/src/app/main.c
@@ -234,9 +234,12 @@ rpmostree_option_context_parse (GOptionContext *context,
       g_print (" Version: '%s'\n", PACKAGE_VERSION);
       if (strlen (RPM_OSTREE_GITREV) > 0)
         g_print (" Git: %s\n", RPM_OSTREE_GITREV);
-      g_print (" Features:\n");
-      for (char **iter = features; iter && *iter; iter++)
-        g_print ("  - %s\n", *iter);
+      if (g_strv_length (features) > 0)
+        {
+          g_print (" Features:\n");
+          for (char **iter = features; iter && *iter; iter++)
+            g_print ("  - %s\n", *iter);
+        }
       exit (EXIT_SUCCESS);
     }

```

?

Reply:
In theory, someone could be looking at those features - that's why they're listed in the first place right?  I don't have an opposition to cleaning it up eventually but it also doesn't seem like a maintenance burden.